src/persist/src/indexed/unsealed.rs

```diff
@@ -396,19 +404,16 @@ impl Iterator for UnsealedSnapshotIter {
                 };
                 match b.recv() {
                     Ok(b) => {
-                        // Reverse the updates so we can pop them off the back
```

Comment:
why did we lose this behavior (and comment about making sure to filter before clone)?

Reply:
iirc, because `ColumnarRecordsIter` doesn't implement `DoubleEndedIterator` -- I think it could though - let me see if I can whip that up

Reply:
nope! that was not the right reason. -- added the `rev` back in, but am not reversing the updates within each `ColumnarRecords` because they are not sorted in any way